cli/tsc/99_main_compiler.js

```diff
@@ -1034,6 +1032,13 @@ delete Object.prototype.__proto__;
           return respond(id, {});
         }
       }
+      case "$didChangeScripts": {
+        const specifiers = args[0];
+        for (const specifier of specifiers) {
+          scriptVersionCache.delete(specifier);
```

Comment:
By the way, this will cause problems where a script won't be invalidated when it changes on the file system via a different tool. This isn't what TypeScript does and I'm not sure we should do this (TypeScript creates a file watcher that invalidates when the file changes on the system for closed documents).